.github/actions/playwright-atomic/action.yml

```
@@ -0,0 +1,18 @@
+name: 'Playwright Atomic'
+description: 'Run Playwright tests for Atomic'
+runs:
+  using: composite
+  steps:
+    - name: Install Playwright Browsers
+      run: npx playwright install --with-deps
+      shell: bash
+    - name: Run Playwright tests
+      run: npx playwright test
+      working-directory: packages/atomic
+      shell: bash
+    - uses: actions/upload-artifact@v4
+      if: always()
+      with:
+        name: atomic-playwright-report
+        path: packages/atomic/playwright-report/
+        retention-days: 30
```

If this does what I think it does, that might be a bit much.

I've personally never had to go back in test reports from more than a day ago. I think a retention of 5 days would be more than sufficient (I'd even go with 1 TBH).

---

It's the artifact on the github run, but yeah, true, let's kick it down
```suggestion
        retention-days: 5
```